src/widgets/DataTable/Cell.js

```diff
@@ -1,36 +1,48 @@
+/* eslint-disable react/forbid-prop-types */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { View, Text, StyleSheet } from 'react-native';
+import { getAdjustedStyle } from './utilities';
 
 /**
  * Renders a cell that displays a string/number value within
  * a react-native `Text` component.
  *
- * @param {string|number} value The value to render in cell
+ * @param {string|number} value     The value to render in cell
+ * @param {Object}        viewStyle Style object for the containing View
+ * @param {Object}        textStyle Style object for the inner Text
  */
-const Cell = React.memo(({ value }) => {
+const Cell = React.memo(({ value, textStyle, viewStyle, width, isLastCell }) => {
   console.log(`- Cell: ${value}`);
+
+  const internalViewStyle = getAdjustedStyle(viewStyle, width, isLastCell);
+
   return (
-    <View style={defaultStyles.cell}>
-      <Text>{value}</Text>
+    <View style={internalViewStyle}>
+      <Text style={textStyle}>{value}</Text>
     </View>
   );
 });
 
+const defaultStyles = StyleSheet.create({
+  viewStyle: {},
+  textStyle: {},
+});
```

Comment:
Seems kind of redundant?

Reply:
Suppose it makes the old just handing an array of styles with overrides work the same?

Reply:
I definitely don't like this, but I'm not sure what to put. I was thinking of adding either our styles, or some basic styles for it to still look like a table before porting them over to `react-native-data-table`?